test/unit/utils.spec.js

```diff
@@ -60,6 +71,65 @@ describe("Util Test", () => {
     expect(toArray(el.childNodes)).to.be.eql(useSlice);
   });
 
+  it("should check 'isCssPropsFromAxes' method", () => {
```

Comment:
Can we use more proper namings for those tests?
Like, in here it says "it should check 'isCssPropsFromAxes' method", but checks what?
Test names should be self-explanatory.
It should explain itself, like what the method/class does, what it reproduces, and more.
For example, in this case, it would be nicer to name it like
```js
describe("isCssPropsFromAxes", () => {
  it("should return true when the CSS props equal default styles of Axes", () => { ... });
  it("should return false when the CSS props not equal default styles of Axes", () => { ... });
  it("should return false when the parameter is null", () => { ... });
});
```

Please don't maintain the repository as-is, the old codes are not always the best.
You can improve where you want.

Reply:
Great, I think I'll be able to change the old test names at the same time as I'm working on this.